packages/rn-tester/js/examples/Playground/RNTesterPlayground.js

```diff
@@ -13,14 +13,59 @@ import type {RNTesterModuleExample} from '../../types/RNTesterTypes';
 
 import RNTesterText from '../../components/RNTesterText';
 import * as React from 'react';
-import {StyleSheet, View} from 'react-native';
+import {StyleSheet, View, TextInput} from 'react-native';
 
 function Playground() {
   return (
     <View style={styles.container}>
       <RNTesterText>
         Edit "RNTesterPlayground.js" to change this file
       </RNTesterText>
+
+      <TextInput
+        style={styles.input}
+        placeholder="Single line - default"
+        placeholderTextColor="black"
+      />
+
+      <TextInput
+        style={styles.input}
+        placeholder="Single line - disableKeyboardShortcuts"
+        disableKeyboardShortcuts
```

Comment:
You should add it to the TextInput example

Reply:
Done - 60e0ea3c3d3aa0a9fff956d092e50094d2f6a72b, a064331.

<details><summary>Screenshot</summary>
<p>

<img width="727" alt="Example" src="https://github.com/user-attachments/assets/62c6bbe8-3552-4b14-947b-f9204faeff06">

</p>
</details>